crates/common/Cargo.toml

```diff
@@ -12,6 +12,9 @@ sparse-merkle-tree = { version = "0.5.2-rc1", default-features = false }
 merkle-cbt = { version = "0.3.0", default-features = false }
 gw-hash = { path = "../hash" }
 thiserror = { version = "1.0", optional = true }
+tokio = { version = "1", features = ["full"] }
```

Comment:
Please move these modules to gw-utils crate.

gw-common is a no-std crate that may be used in Rust contract.